configure.ac

```diff
@@ -141,7 +141,9 @@ AC_SUBST(GLUSTER_BLOCKD_WORKDIR)
 
 if test "x$prefix" = xNONE; then
    test $libexecdir = '${prefix}/libexec' && localstatedir=$ac_default_prefix/libexec
-   libexecdir=/usr/libexec
+   libexecdir=usr/libexec
```

Comment:
@lxbsz did you intentionally removed '/' before usr/libexec ?

Reply:
Ah, sorry. This is by mistake.
Fixed it.
Thanks.